.eslintrc.js

```diff
@@ -156,6 +156,7 @@ module.exports = {
     "unicorn/prevent-abbreviations": "off",
     "import/order": "off",
     "@typescript-eslint/no-empty-function": "off",
+    "unicorn/no-array-reduce": "off",
```

Comment:
Feel free to push back on this - I added this in because I used a reduce and I don't love adding one-off eslint ignores. When this rule was discussed with the team previously, no one seemed to care too much about using reduces, so I decided to add it in.